.travis.yml

```diff
@@ -34,6 +34,9 @@ jobs:
     - stage: rpms
       name: "Build Fedora 31 RPMs"
       script: ./tests/build-and-install-rpms.sh fc31 dockerfiles/Fedora31.dockerfile
+    - stage: packages
```

Comment:
Can we name this stage `debs` instead of `packages`?

Reply:
Fine by me. Will change after finishing dinner.